deploy/setup.sh

```diff
@@ -0,0 +1,280 @@
+   echo "installing mongodb-enterprise=3.0.11 mongodb-enterprise-server=3.0.11 mongodb-enterprise-shell=3.0.11 mongodb-enterprise-mongos=3.0.11 mongodb-enterprise-tools=3.0.11"
+   result=$(sudo apt-get install -y mongodb-enterprise=3.0.11 mongodb-enterprise-server=3.0.11 mongodb-enterprise-shell=3.0.11 mongodb-enterprise-mongos=3.0.11 mongodb-enterprise-tools=3.0.11)
+   if grep -q "Unable to lacate" <<< $result; then
+       echo "Could not install mongodb, pls run this script again or install mongodb manually"
+       exit
+   fi
+   echo "mongodb-enterprise hold" | sudo dpkg --set-selections
+   echo "mongodb-enterprise-server hold" | sudo dpkg --set-selections
+   echo "mongodb-enterprise-shell hold" | sudo dpkg --set-selections
+   echo "mongodb-enterprise-mongos hold" | sudo dpkg --set-selections
+   echo "mongodb-enterprise-tools hold" | sudo dpkg --set-selections
+   service mongod start
+   cd /home/opentech/ && python open-hackathon-server/src/setup_db.py
+}
+
+get_dependency_for_guacamole() {
+    echo "solve dependency software for guacamole"
+    result=$(sudo service guacd restart)
+    if grep -q "SUCCESS" <<< $result; then
+        echo "guacamole is installed!"
+        return
+    fi
+    echo "installing libcairo2-dev"
+    result=$(sudo apt-get install -y libcairo2-dev)
+    if grep -q "Unable to lacate" <<< $result; then
+        echo "Could not install dependancy software for guacamole, pls install guacamole manually"
+        exit
+    fi
+    echo "installing libjpeg62-dev libpng12-dev libossp-uuid-dev"
+    result=$(sudo apt-get install -y libjpeg62-dev libpng12-dev libossp-uuid-dev)
+    if grep -q "Unable to lacate" <<< $result; then
+        echo "Could not install dependancy software for guacamole, pls install guacamole manually"
+        exit
+    fi
+    echo "installing libfreerdp-dev libpango1.0-dev libssh2-1-dev libtelnet-dev"
+    result=$(sudo apt-get install -y libfreerdp-dev libpango1.0-dev libssh2-1-dev libtelnet-dev)
+    if grep -q "Unable to lacate" <<< $result; then
+        echo "Could not install dependancy software for guacamole, pls install guacamole manually"
+        exit
+    fi
+    echo "installing libvncserver-dev libpulse-dev libwebp-dev libssl-dev libvorbis-dev"
+    result=$(sudo apt-get install -y libvncserver-dev libpulse-dev libwebp-dev libssl-dev libvorbis-dev)
+    if grep -q "Unable to lacate" <<< $result; then
+        echo "Could not install dependancy software for guacamole, pls install guacamole manually"
+        exit
+    fi
+}
+
+install_and_config_guacamole() {
+    result=$(sudo service guacd restart)
+    if grep -q "SUCCESS" <<< $result; then
+        echo "guacamole is installed!"
+        return
+    fi
+    echo "installing guacamole"
+    # install and Configure guacamole
+    cd /home/opentech && rm -f download && rm -rf guacamole-server-0.9.9
+    wget http://sourceforge.net/projects/guacamole/files/current/source/guacamole-server-0.9.9.tar.gz/download
```

Comment:
You probably need to check the existence of the file to avoid duplicate download